src/state/shellscripts/bwrap.sh

```diff
@@ -53,11 +54,21 @@ add_sys_mounts() {
     done
 }
 
-# remove some unusual paths (/nix/stored and /rw/usrlocal )
-# use OPAM_USER_PATH_RO variable to add them
-# the OPAM_USER_PATH_RO format is the same as PATH
-# ie: export OPAM_USER_PATH_RO=/nix/store:/rw/usrlocal
-add_sys_mounts /usr /bin /lib /lib32 /lib64 /etc /opt /home /var /tmp
+# NOTE: The list of moint-points to avoid is always to be sync'd with the list at the top of the file
+# It is due to a limitation of bubblewrap that we have to mount the directories one by one
+# See https://github.com/containers/bubblewrap/issues/413
+for dir in $(ls -a /); do
```

Comment:
Using `ls` in shell scripts is generally discouraged, there may be lots of output options enabled locally.
I would go with something like:
```bash
for dir in /*; do
    if [ ! -d "$dir" ]; then continue; fi
    case "$dir" in
        /proc|/dev|/run|/opam-tmp) ;;
        *) add_sys_mounts "$dir";;
    esac
done
```

The difference is that this wouldn't mount hidden root directories... could that be an issue ?

Reply:
> there may be lots of output options enabled locally

Unless the user is purposfully using `source sandbox-script.sh` or called using `bash -l`/other non-interactive/login options, user changes (`/etc/profile`, `~/.bashrc`, …) are disabled when calling scripts so this should never happen.

Reply:
Actually I a solution to get the best of both world (use `/*` and get the hidden files): `shopt -s dotglob`.
I'll fix that right now.